getting-started/optional_installation_settings.md

```diff
@@ -0,0 +1,220 @@
+# Optional Installation Settings
+## Testing Support Setup
+
+Some of our tests we present in the guide examples utilize non-standard requirements that can be appended to your local rails helper. These additions will help provide an error free learning experience! Let's start with our gemfile to make sure we have required dependencies:
+
+{% code title="Gemfile" %}
+```ruby
+group :development, :test do
+  # Preferred testing suite which we use in our Solidus guides
+  gem 'rspec-rails'
+  # Setup 'factories' or prebuilt objects to expedite testing
+  gem 'factory_bot_rails' '~> 4.8'
+  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
+  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+end
+
+#...
+
+group :test do
+# Adds support for Capybara system testing and selenium driver
+  gem 'capybara', '>= 3.26'
+  # Easy installation and use of web drivers to run system tests with browsers
+  gem 'webdrivers'
+  # Scrubs database clean between tests
+  gem 'database_cleaner-active_record'
+```
+{% endcode %}
+
+Let's go over installation of these gems and why we are suggesting them!
+### RSpec
+RSpec is highly versatile testing framework that we utilize for Solidus and in our guide examples. To install it for your local project, simply Update your bundle and `Gemfile.lock` then install RSpec base files with the following commands in your shell:
```

Comment:
```suggestion
RSpec is a highly versatile testing framework that we utilize for Solidus and in our guide examples. To install it for your local project, simply update your bundle and install RSpec base files with the following commands in your shell:
```

I think it's weird to talk about updating `Gemfile.lock` as it seems to suggest to do it manually. Do you agree?